### PR TITLE
Log server version at startup

### LINKED ISSUE #159: Log ark version when server starts

### PR DESCRIPTION
**Pull request by ncdc**
Fixes #159

Signed-off-by: Andy Goldstein <andy.goldstein@gmail.com>

### CODE REVIEW

File: pkg/cmd/server/server.go [hunk 1]
```diff
@@ -83,6 +84,7 @@ func NewCommand() *cobra.Command {
 			logrus.Infof("setting log-level to %s", strings.ToUpper(logLevel.String()))
 
 			logger := newLogger(logLevel, &logging.ErrorLocationHook{}, &logging.LogLocationHook{})
+			logger.Infof("Staring Ark server %s", buildinfo.FormattedGitSHA())
```
Comment (skriss) [inline pkg/cmd/server/server.go]: "Staring" -> "Starting"

Comment (ncdc) [reply pkg/cmd/server/server.go]: Fixed

**Comment by skriss** [comment]
are you intentionally not logging the actual Version?

**Comment by ncdc** [comment]
Not really. But if the version is "master" I'd rather see the git details. And this should work for both tags and branches.

**Comment by skriss** [comment]
OK.

**Pull request merged**